forms/jqm/elements/choiceexpanded.js

```diff
@@ -21,24 +21,35 @@ define(function (require) {
       return ChoiceElementView.prototype.remove.call(this);
     },
 
+    renderLabel: function () {
+      if (!this.$label) {
+        this.$label = $('<legend></legend>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
Is there a way to accomplish the desired result without using `jQuery.fn.find()`? That'll cause a complete traversal of the sub-tree (including all the `<options />`), which admittedly won't be as bad as for sub-records, but is still extra work that might be avoided.